crates/fluvio-version-manager/src/common/workdir.rs

```diff
@@ -70,10 +66,10 @@ mod tests {
     }
 
     #[test]
-    fn test_fvm_pkgset_path() {
-        let fvm_pkgset_path = fvm_pkgset_path().expect("Failed to get fvm pkgset path");
+    fn test_fvm_versions_path() {
+        let fvm_version_path = fvm_versions_path().expect("Failed to get fvm pkgset path");
         let fvm_path = fvm_workdir_path().expect("Failed to get fvm path");
 
-        assert_eq!(fvm_pkgset_path, fvm_path.join(FVM_PACKAGES_SET_DIR));
+        assert_eq!(fvm_version_path, fvm_path.join(FVM_VERSIONS_DIR));
     }
```

Comment:
See the same comments from the fluvio bin directory.  Add actual assertion.  Don't just assume code does the right thing.